addons/l10n_es_edi_sii/__init__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from . import models
+from odoo import api, SUPERUSER_ID
+
+
+def _setup_tax_type(env):
+    companies = env['res.company'].search([('partner_id.country_id.code', '=', 'ES')])
+    chart_templates = companies.chart_template_id
+    current_chart_templates = chart_templates
```

Comment:
`current_chart_templates` being plural is weird when it looks like there's only ever one.

Reply:
companies.chart_template_id can be different chart_templates.  Would not it be more logical to use mapped(' in those cases??